bikeshed/boilerplate/whatwg/computed-metadata-RD.include

```diff
@@ -0,0 +1,4 @@
+{
+  "Title": "[SPECTITLE] Standard Review Draft [ISODATE]",
```

Comment:
It seems somewhat friendlier to use "May 2018" in the title.

Reply:
@tabatkins can we get a macro that turns the defined Date: metadata into a human-readable "May 2018" format?

Reply:
Done. Didn't know whether you wanted short or full month names, so I added both - `[DATE-MY]` gives short month, `[DATE-MMY]` gives full month.

(And for consistency, `[DATE]` now has `[DATE-DMMY]` as an alias.)